app/models/peep.rb

```diff
@@ -0,0 +1,15 @@
+require 'data_mapper'
+require 'dm-postgres-adapter'
+
+class Peep
+
+  include DataMapper::Resource
+
+  property :id,     Serial
+  property :text,  String
+
+end
+
+DataMapper.setup(:default, ENV['DATABASE_URL'] || "postgres://localhost/chitter_#{ENV['RACK_ENV']}")
```

Comment:
these should not go here - you could extract the datamapper setup stuff into a separate file and then require it in the main server file.